src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,27 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { Movie } from '../../types/Movie';
```

Comment:
you haven't added this file to commit, please add it in next commit

Reply:
Fixed